packages/components/stories/auto-height-wrapper/README.md

```diff
@@ -0,0 +1,46 @@
+# Auto Height Wrapper
+
+Renders a wrapper to control the height of elements. `AutoHeightWrapper` does not allow the child component's height to be greater than the client's screen size and makes sure that the children is always visible.
+
+
+## Usage
+
+```jsx
+import AutoHeightWrapper from 'Components/auto-height-wrapper';
+
+const DummyComponent = props => (
+    <AutoHeightWrapper default_height={300} height_offset={50}>
+        {({ setRef, height }) => (
+            <div ref={setRef} style={{height}}/>
+        )}
+    </AutoHeightWrapper>
+);
+```
+
+## Props
+
+| Name            | Type            | Default          | Description                                             |
+| --------------- | --------------- | ---------------- | ------------------------------------------------------- |
+| default_height  | {number}        | null             | Default height of the element.                          |
```

Comment:
If it's a number why we set the default to `null`?

Reply:
There is no default value for `default_height`